orbit/pkg/platform/platform_windows.go

```diff
@@ -328,9 +328,10 @@ func hardwareGetSMBiosUUID() (string, error) {
 // the actual SMBIOS hardware interface.
 func GetSMBiosUUID() (string, UUIDSource, error) {
 	// It attempts first to get the UUID from WMI
+	log.Debug().Msg("running wmiGetSMBiosUUID to retrieve UUID")
 	uuid, err := wmiGetSMBiosUUID()
```

Comment:
General question about the agent's ID. (Regardless of this PR)
What happens if in the future we have customers that run multiple agents on VMs on the same HW?
Will it not get the same UUID?

Reply:
Good question. 

Yes. That scenario is currently not supported by Orbit. See #8021.